tasks/populate-release-notes-images/tests/test-populate-release-notes-images-multiple-images.yaml

```diff
@@ -114,7 +116,7 @@ spec:
               test $(jq -r '.architecture' <<< $image1arch1) == "amd64"
               test $(jq -r '.containerImage' <<< $image1arch1) == "registry.redhat.io/product/repo@sha256:abcdefg"
               test $(jq -r '.purl' <<< $image1arch1) == \
-                "pkg:oci/repo@sha256%3Aabcdefg?arch=amd64&repository_url=registry.redhat.io/product"
+                "pkg:oci/repo@sha256%3Aabcdefg?arch=amd64&repository_url=registry.redhat.io/product&tag=9.4.0-1723436855"
```

Comment:
You also need to add the new tags to the test 2 lines below. The test currently fails on that. The same applies for `image1arch2` as well.